### PR TITLE
Pod Anti Affinity Docs are inconsistent

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1600441

### CODE REVIEW

**Comment by mburke5678** [comment]
@aveshagarwal 
Removed the DC example from Manage Nodes and added Pod Config from Pod Affinity topic. Is this change appropriate?

**Comment by aveshagarwal** [comment]
@mburke5678 lgtm.

**Comment by mburke5678** [comment]
@mdshuai Please take a look at these changes.

**Comment by mdshuai** [comment]
> @mdshuai Please take a look at these changes.

@wjiangjay will help review it.

**Comment by wjiangjay** [comment]
LGTM

**Comment by mburke5678** [comment]
@aveshagarwal From your comment in the BZ: "The above pod anti-affinity configuration is not valid since 3.5/3.6." Should this change go into 3.5+ or 3.6+?
It appears that there was a change in 3.6. based on the release notes.
https://docs.openshift.com/container-platform/3.6/release_notes/ocp_3_6_release_notes.html#ocp-36-pod-affinity-and-anti-affinity

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -415,32 +415,34 @@ anti-affinity can be set to either required or preferred.
...
+            - key: docker-registry <4>
+              operator: In <5>
+              values: 
+              - default
+          topologyKey: kubernetes.io/hostname
+----
+
+<1> Stanza to configure pod anti-affinity.
+<2> Defines a preferred rule.
+<3> Specifies a weight for a preferred rule. The node that with highest weight is preferred.
```
Comment (kalexand-rh) [inline admin_guide/manage_nodes.adoc]: "The node that with highest weight is preferred." doesn't make sense.
I haven't read any of the context, but I think it should say something like "The pod runs on the node with the highest weight" or "The pod applies the rule with the largest weight."

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -415,32 +415,34 @@ anti-affinity can be set to either required or preferred.
...
+<4> The key and value (label) that must be matched to apply the rule.
```
Comment (kalexand-rh) [inline admin_guide/manage_nodes.adoc]: This annotation confused me until I read annotation 5. I think it should be something like "Description of the pod label that determines when the anti-affinity rule applies. Specify a key and value for the label."
( also might attach it to the line that contains matchExpression or give annotations for both the key and value lines.)

File: admin_guide/manage_nodes.adoc [hunk 3]
```diff
@@ -415,32 +415,34 @@ anti-affinity can be set to either required or preferred.
...
+<5> The operator represents the relationship between the label on the existing pod and the set of values in the `matchExpression` parameters in the specification for the new pod.  Can be `In`, `NotIn`, `Exists`, or `DoesNotExist`.
```
Comment (kalexand-rh) [inline admin_guide/manage_nodes.adoc]: There's an extra space before "Can."
s/Can be/Specify

**Review by kalexand-rh — COMMENTED** [review]
Couple of requests and suggestions. :stars:

File: admin_guide/manage_nodes.adoc (hunk 1 above)
Comment (mburke5678) [reply admin_guide/manage_nodes.adoc]: @kalexand-rh I think the meaning is right. Rules can be required to be followed or followed unless it is not possible (preferred). This setting is only for a preferred rule. If the wording can be better, let me know!

Comment (kalexand-rh) [reply admin_guide/manage_nodes.adoc]: After our chat, I think we should revisit this after 3.10

s/that with/with the

**Pull request merged**